### ISSUE TITLE
Browser layout issues reported in Firefox, Chrome, and Safari

### ISSUE THREAD

**Issue by bryanwillis** [OakbrookEstates/website#2]
These browsers need to be checked and made sure that they either support the modern css styling technologies or have some sort of fallback if they're not working.  

Most likely these issues are browser bugs/quirks in browsers that don't fully support modern flexbox styling or they are just missing the prefixes.

**Comment by bryanwillis** [comment]
I've made sure the prefixes were all there and ran some browser tests.

Everything appears to be working well (see test results below). If you notice anything that the tests aren't picking up, let me know otherwise I'll close this issue as resolved.

**Test Results (chrome, firefox, safari, ipad, android, ie):**
https://app.crossbrowsertesting.com/public/i3a7f2755d98e11b/screenshots/zfaa92c0f8c1b932a8fb

**Comment by bryanwillis** [comment]
I checked all the computers at MBI Wednesday after we talked using Internet Explorer and checked again at the library today on a public computer. Based on everything I'm seeing the results were identical as what I see on Google Chrome.

I also had the same results on Safari using 3 different Macs (utilizing the latest version of Safari on Joe's laptop, version 8.03 on my mac, as well as version 6.2 ([see results](https://app.crossbrowsertesting.com/public/i3a7f2755d98e11b/screenshots/zfaa92c0f8c1b932a8fb/z09ee599bc69a8a408f4?all_public=true&type=fullpage)). 

At this point unfortunately there isn't anything I can do without being able to have a better understanding on what you're seeing on Internet Explorer and Safari. [Separate screenshots of every browser tested can be viewed here](https://app.crossbrowsertesting.com/public/i3a7f2755d98e11b/screenshots/zfaa92c0f8c1b932a8fb?size=small&type=windowed) if you're interested in the results. They all appear to look the same to me and the time and phone number both show in every browser as far as I can tell, but forgive a I might be missing something obvious or not understanding the issue correctly! 

To move forward with you can take screenshots of what you're seeing. Here the instructions in case you haven't taken a screenshot before on a mac or pc.  

## Screenshot Instructions

**Windows 10 or Windows 8 Screenshot**
Pressing both the "_Windows + Print Screen key_" at the same time will capture the entire screen. This image will automatically be saved to a Screenshot folder inside of the Pictures library.

**MAC / SAFARI**
Press "_Shift + Command + 4" all at the same time. The screenshot should save to the desktop by default.

Once you take the screenshot, just drag and drop it to the comment box below and comment on what version of IE and/or safari version is being used in the screenshot you uploaded. To check the version on IE open it up and click the gear Icon (usually in the top right corner of the browser I believe). 

## On safari here's how to check the version.

<img width="1440" alt="screen shot 5" src="https://cloud.githubusercontent.com/assets/4042621/18798228/16c33c70-8197-11e6-9631-07190c38dac3.png">

If this seems too complicated, I can also have you share your screen with me using [google hangouts](https://hangouts.google.com/). I'm out at a charity event all day tomorrow, but I'm available all day today, Sunday afternoon, or we could set up a call next week to do this. It's very easy to do and would only take 15 minutes at the most if you'd rather go this route.

Anyway, let me know or give me a call. The sooner I can see whats going on the sooner I can get this fixed for you.

**Comment by bryanwillis** [comment]
# Reopening this issue. 

### Below are updated screenshots of each browser with no seen issues.

### And [**here is a live video recording**](https://youtu.be/dx0RzejuHyw ) of every device being tested for http://oakbrookestates.com homepage




## **Chrome 55**

![bs_macsie_chrome_55 0](https://cloud.githubusercontent.com/assets/4042621/21539060/82b92ba2-cd68-11e6-8469-76cd09827834.jpg)


<hr>

## **Firefox 50**

![bs_macsie_firefox_50 0](https://cloud.githubusercontent.com/assets/4042621/21539061/82c11b96-cd68-11e6-9f6e-46df30425547.jpg)

<hr>

## **Samsung Galaxy Note**

![bs_realdroid_mobile_samsung galaxy note 4-4 4-1440x2560](https://cloud.githubusercontent.com/assets/4042621/21539058/826c8b1c-cd68-11e6-9fdb-a93476eb4322.jpg)

<hr>

## **iphone 6s**

![bs_realios_mobile_iphone 6s-9 0](https://cloud.githubusercontent.com/assets/4042621/21539085/b8bd562e-cd68-11e6-955d-677411a4e79e.jpg)

<hr>

## **IE 11 on Windows 10**

![bs_win10_ie_11 0](https://cloud.githubusercontent.com/assets/4042621/21539059/829d5530-cd68-11e6-847f-d3396ae0f8a8.jpg)

**Comment by bryanwillis** [comment]
As per meeting 1/11 we concluded the browser layout issues could not be reproduced on any modern os's browsers. 

There is however a strange bug in **IE11** on **Windows 7** I narrowed down that occurs sporadically causing the phone number in the header to be invisible. The bug causes certain fonts to disappear when using Google fonts with `font-feature-settings: "kern" 1` set. 

_Details of the bug can be found [here](https://connect.microsoft.com/IE/feedbackdetail/view/831964)_.

While I'm unable to offer too much support for Windows 7 I was able to come up with a workaround by using css and targeting IE11 specifically for Windows 7  and applying a sort of "hack". This is probably not necessary considering this bug occurs to less than 1% of the browser/device marketshare, but I wanted to implement it since you are in using this browser.

This issue only occurs under very specific circumstances and it's important to make sure to stay up to date with Microsoft security patches and fixes or try [reinstalling the updated operating system](https://www.microsoft.com/en-us/download/Internet-Explorer-11-for-Windows-7-details.aspx). I also suggest trying "[Safe Mode](http://www.dell.com/support/article/us/en/19/SLN265764/EN#difference)" whenever an issue like this arrises using a legacy operating system. 

I tested my pseudo fix in a virtual machine and all appears to be working, but then again it gave a false negative last time. I suggest testing out each page to ensure it works properly for you now and **let me know**. Worst case scenario, I remove the fix and either change the font or just have it not show the phone number on the IE11 for Windows 7.
